DocAggregatorDotnetCore/TomAggregator.cs

```diff
@@ -38,15 +33,22 @@ public void DoSomeWork()
 
             Directory.CreateDirectory(outputDirectoryPath);
 
-            //Console.WriteLine("New folders to go in: " + outputDirectoryPath);
+            //make array of property refs
+            var ctx = new RiggsTestContext();
+            var listOfDocs = ctx.PropDocs.Select(p => p).ToList();
```

Comment:
You don't need the Select here as the default is to bring everything back.

Reply:
I'm not sure if you've read about Change Tracking in Entity Framework yet, but if you aren't planning to update any of the results coming back from this call, you could use .AsNoTracking() to opt out the Change Tracking behaviour, which will reduce memory footprint etc.

Reply:
Fixed!